e2e/app/page/workspace-about-page.ts

```diff
@@ -20,26 +18,20 @@ export default class WorkspaceAboutPage extends WorkspaceBase {
     return true;
   }
 
-  async findUserInCollaboratorList(username: string): Promise<WorkspaceAccessLevel> {
-    // At least one collab should eventually render, i.e. the current user.
-    const collabXPath = '//*[starts-with(@data-test-id,"workspaceUser-")]';
-    await this.page.waitForXPath(collabXPath, { visible: true });
+  // Returns map of key: Access Role, value: User Email.
```

Comment:
This would be more reliable if it worked in the other direction.  For example, what if this workspace had 2 readers?